crates/nu-command/tests/commands/utouch.rs

```diff
@@ -168,6 +256,31 @@ fn change_modified_and_access_time_of_file_to_today() {
     })
 }
 
+#[test]
+fn change_modified_and_access_time_of_files_matching_glob_to_today() {
```

Comment:
I think it's fine to keep just this test and get rid of the `utouch -a *.txt`, `utouch -m *.txt`, and `utouch -c *.txt` tests. And even in this test, you can use `utouch *.txt` rather than `utouch -a -m *.txt`. We can probably trust that the flags will continue working, since your changes don't touch that.

Let me know if you have reason to leave those other tests in, though.